pages/national_resources/no_resources.md

```diff
@@ -242,6 +242,21 @@ Sikt is a Norwegian governmental agency that provides [(sensitive) personal data
 
 NRIS, a collaboration between Sigma2 (a subsidiary of Sikt) and four universities, gives [technical and administrative support](https://www.sigma2.no/user-support) to researchers in Norway on large-scale data storage and high-performance computing.
 
+
+### Communities for research data management support personell
+
```

Comment:
Maybe an introductory sentence could be useful here f.ex something along the lines of "Norway has several RDM communities that are created to bring together data management support staff/personnel through knowledge-exchange activities and discussions on specific DM topics" (now that I have read 'personell', I am not sure which spelling is correct. I was only familiar with 'personnel')

Reply:
Nowegian spelling vs English fixed now